week-3/Homework/extra/1-card-validator.js

```diff
@@ -0,0 +1,26 @@
+function validateCreditCard(cardno){
```

Comment:
This is looking great, but we're missing one of the checks - that all of the digits aren't all the same. Could you add a check for that, too?